iaso/api/forms.py

```diff
@@ -392,6 +392,9 @@ def get_queryset(self, mobile=False):
 
         queryset = queryset.order_by(*order)
 
+        # Ensure duplicates are removed after all joins and annotations
+        queryset = queryset.distinct()
```

Comment:
I just wanted to point out that [using `distinct()` without a specific field is slower](https://docs.djangoproject.com/en/4.2/ref/models/querysets/#django.db.models.query.QuerySet.distinct) in general:

> For a normal `distinct()` call, the database compares each field in each row when determining which rows are distinct. For a `distinct()` call with specified field names, the database will only compare the specified field names.

However, we are mixing aggregation (`annotate`) and `distinct` here… So calling `distinct` with specific field names [would cause a lot more headaches](https://code.djangoproject.com/ticket/32663).

Let's stick with `distinct()`.